.grit/patterns/python/cleaned_data_vs_post.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Use `$FORM.cleaned_data[]` instead of `request.POST[]` after `form.is_valid()`
+---
+
+Use `$FORM.cleaned_data[]` instead of `request.POST[]` after `form.is_valid()` has been executed to only access sanitized data.
+
+### references
+- https://docs.djangoproject.com/en/4.2/ref/forms/api/#accessing-clean-data
+
+tags: #fix, #best-practice
```

Comment:
Move to frontmatter

Reply:
updated

Reply:
Remove it here though... it shouldn't be in both places.